Cargo.toml

```diff
@@ -63,5 +63,11 @@ members = [
 missing_debug_implementations = "warn"
 rust_2018_idioms = "warn"
 single_use_lifetimes = "warn"
+unexpected_cfgs = { level = "warn", check-cfg = [
```

Comment:
seems this warn isn't avoidable, when locally-developing crossbeam itself  with any toolchain with the support of `Cargo.toml`'s `lints.rust` and `< nightly-2024-05-19`..:

```
$ RUSTUP_TOOLCHAIN=1.78.0-x86_64-unknown-linux-gnu cargo check
warning: /home/ryoqun/work/crossbeam/Cargo.toml: unused manifest key: workspace.lints.rust.unexpected_cfgs.check-cfg
   Compiling crossbeam-utils v0.8.19 (/home/ryoqun/work/crossbeam/crossbeam-utils)
```

Reply:
Btw, we are working on making this warning go away in the current beta with https://github.com/rust-lang/cargo/pull/13925, so that when it reaches stable, the warning would already be gone by the previous version.